.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+readme.md
```

Comment:
- [ ] Good job so far but I noticed that you ignore your `readme.md` file, as required for this project you should ignore `test.md` and `node_modules` only, kindly change this for you to move forward :wink: :wink: 
- [ ] Your Readme should be visible on your repository